moveit_experimental/moveit_jog_arm/src/jog_cpp_interface.cpp

```diff
@@ -153,6 +157,22 @@ void JogCppInterface::stopMainLoop()
   stop_requested_ = true;
 }
 
+void JogCppInterface::pause()
+{
+  if (jog_calcs_)
+    jog_calcs_->pauseOutgoingJogCmds();
+
+  paused_ = true;
+}
```

Comment:
nit, combine the two:
```suggestion
void JogCppInterface::pause(bool pause)
{
  if (jog_calcs_)
    jog_calcs_->pauseOutgoingJogCmds(pause);

  paused_ = pause;
}
```